Installing yay broken when GO111MODULE=on

#### Affected Version

```
$ yay -V
yay v8.1115 - libalpm v11.0.1
```

This is after I worked around the problem and upgraded. It's more about the build than the version of yay though, so it will still happen on this version upgrading to a newer one or when attempting to rebuild.

#### Issue

#### Steps to reproduce

1. `export GO111MODULE=on` – enable [Go modules] (even if we're in GOPATH)
2. `yay -Su yay` (or similar: attempt to upgrade yay from an older version anyways)

```
…
==> Starting build()...
go build -v -ldflags '-s -w -X main.version=8.1115' -o yay
go: cannot determine module path for source directory /home/sam/.cache/yay/yay/src/yay-8.1115 (outside GOPATH, no import comments)
make: *** [Makefile:46: build] Error 1
```

Adding an import directive (eg. `package main // import "github.com/Jguer/yay"`) should fix this. Alternatively, switching to go modules from dep (or adding a step to the build to generate go mod files from the dep files if you're not ready to switch entirely) would fix it.

[Go modules]: https://golang.org/doc/go1.11#modules

--- a/main.go
+++ b/main.go
@@ -1,4 +1,4 @@
-package main
+package main // import "github.com/Jguer/yay"
 
 import (
 	"encoding/json"

